### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by zgojcic** [zjhthu/OANet#10]
Hi, 

I was just wondering if you could please clarify some inconsistencies between the paper and the provided implementation. Specifically, in the paper you state that you use learning rate of 10^-4 and that the weight alpha is 0.1 after 20k iterations. However, in the provided config the weight of the essential loss is 0.5 and the learning rate is 10^-3. 

Thanks
Zan

**Comment by zjhthu** [comment]
A sad story... I just find there is a mistake in our paper, the learning rate is 1e-3 as shown in our code, all our experiments use this lr.
The statement about the essential loss is also not clear. The weight of the essential loss is 0.1 for L2 loss but 0.5 for geometry loss (section 4.4). We use geometry loss as the default loss because it is better, so the default weight is 0.5.
Thanks for the question!

**Comment by zgojcic** [comment]
Thanks for the quick response and no worries, I just wanted to double check this.
